api/src/routes.ts

```diff
@@ -86,6 +86,12 @@ export const routes: IRoute[] = [
     { path: "/stats/:network", method: "get", folder: "stats", func: "get", sign: true },
     { path: "/market/:currency", method: "get", folder: "market", func: "get", sign: true },
     { path: "/did/:network/:did/document", method: "get", folder: "identity/resolution", func: "get" },
+    {
+        path: "/stardust/did/:network/:did/document",
+        method: "get",
+        folder: "identity/resolution-stardust",
```

Comment:
I agree, I didn't put them there first because we wanted to keep everything around identity separated from the rest when possible. But I don't think that's important anymore.